packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -294,13 +285,7 @@ class AccountSwitcher extends React.Component {
         const demo_accounts = (
             <div className='acc-switcher__list-wrapper'>
                 <AccountWrapper
-                    header={
-                        <AccountTitle
-                            title={localize('Deriv Accounts')}
-                            is_eu={this.props.is_eu}
-                            onIconClick={this.showAccountTypesModal}
-                        />
-                    }
+                    header={<span>{localize('Deriv Accounts')}</span>}
```

Comment:
Do these need to be inside a `span` specifically? Or is it just that they need to be of type `node` and not just `string`? If they just need to be a `node` and doesn't matter to specifically be `span`, can we use `<Localize />` instead then for all of them?